src/Widget/Twig/TwigPreprocessor.php

```diff
@@ -784,6 +784,33 @@ protected function isUitpasEvent(Event $event)
         return false;
     }
 
+    /**
+     * Check if event is considered an "Museumpas" event.
+     *
+     * @param \CultuurNet\SearchV3\ValueObjects\Event $event
+     * @return bool
+     */
+    protected function isMuseumpasEvent(Event $event)
```

Comment:
Nit: It's better to go with `private` by default and only use `protected` if necessary. That way refactoring becomes easier in the future if ever needed.

Reply:
Alright, I will keep it in mind!